Tasks/VsTest/task.json

```diff
@@ -47,11 +48,24 @@
     "inputs": [
         {
             "name": "testAssembly",
-            "type": "string",
+            "type": "multiLine",
             "label": "Test Assembly",
-            "defaultValue": "**\\*test*.dll;-:**\\obj\\**",
+            "defaultValue": "**\\*test*.dll\n!**\\obj\\**",
             "required": true,
-            "helpMarkDown": "Test binaries to run tests on.  Wildcards can be used.  For example, `**\\*test*.dll;-:**\\obj\\**` for all dlls with test in name while excluding files in any sub-directory named obj.",
+            "helpMarkDown": "Test binaries to run tests on. Supports multiple lines of minimatch patterns.",
```

Comment:
what are the minimatch patterns? ref link to customers? They don't know what's minimatch.
It should be clear in help description how to include how to exclude